os/board/common/Kconfig

```diff
@@ -66,6 +66,13 @@ config ARCH_BOARD_HAVE_SECOND_FLASH
 if ARCH_BOARD_HAVE_SECOND_FLASH && (NFILE_DESCRIPTORS != 0)
 comment "Second Flash Partition Options"
 
+config FLASH_SECOND_START_ADDR
+    hex
+    default 0xA000000
```

Comment:
It's right that user should not modify this. But this config should be changed by chip selection. Then how can we change this for other chip?

Reply:
done